src/js/components/Form/stories/ArrayOfFormFields.js

```diff
@@ -0,0 +1,131 @@
+import React, { useState } from 'react';
+
+import { grommet, Box, Form, Grommet, Button, FormField } from 'grommet';
+import { Add, Trash } from 'grommet-icons';
+
+export const ArrayOfFormFields = () => {
+  const [values, setValues] = useState({
+    name: '',
+    phones: [{ number: '', ext: '' }],
+  });
+
+  const addPhone = () => {
+    const newPhone = { number: '', ext: '' };
+    const newPhones = [...values?.phones, newPhone];
+    setValues({
+      ...values,
+      phones: newPhones,
+    });
+  };
+
+  const removePhone = (index) => {
+    if (values.phones && values.phones.length > 0) {
+      setValues({
+        ...values,
+        phones: values.phones.filter((v, _idx) => _idx !== index),
+      });
+    }
+  };
+
+  const handleFormChange = (newFormState) => {
+    console.log({ newFormState });
+    if (newFormState) {
+      setValues(newFormState);
+    }
+  };
+  let PhoneNumberGroup = null;
+  if (values.phones !== undefined) {
+    PhoneNumberGroup = values.phones.map((phone, index) => (
+      <Box
+        key={`${phone.number + index}`}
```

Comment:
```suggestion
        key={`${index}`}
```
phone number hasn't been entered yet and is subject to change, it shouldn't be used in the key. Using it as part of the key is causing an issue when typing into the phone number field you can only type one number at a time.